src/NUnitFramework/tests/TestContextTests.cs

```diff
@@ -233,6 +237,123 @@ public void TestContextStoresFailureInfoForOneTimeTearDown()
             Assert.That(fixture.Message, Is.EqualTo(TestResult.CHILD_ERRORS_MESSAGE));
             Assert.That(fixture.StackTrace, Is.Null);
         }
+#if ASYNC
+
+#if PORTABLE
+        [Ignore("Not implemented yet as TestExecutionContext uses TLS in portable implementation")]
```

Comment:
My first thought was that you should use a PlatformAttribute here, but portable isn't really a platform but a combination of platforms. Not sure how to deal with that any better than you have done. 😞